### PR TITLE
Release change for cred scan

### PR DESCRIPTION
**Pull request by arvindsingh-cloud**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.Release.ps1 [hunk 1]
```diff
@@ -85,8 +84,69 @@ class Release: SVTBase
...
+                        "Found credentials in release definition. Variables names: $varList" );
+                    }
+                else {
+                    $controlResult.AddMessage([VerificationResult]::Passed, "No credentials found in release definition.");
+                }
+                $patterns = $null;
+                }
+            }
+            catch {
+                $controlResult.AddMessage([VerificationResult]::Manual, "Could not evaluated release definition.");
```
Comment (abhaydaga) [inline src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.Release.ps1]: evaluate

File: src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.Release.ps1 [hunk 2]
```diff
@@ -85,8 +84,69 @@ class Release: SVTBase
...
+                            break  
+                            }
+                        }
+                    } 
+                    }
+                    if($noOfCredFound -gt 0)
+                    {
+                        $varList = $varList | select -Unique
+                        $controlResult.AddMessage([VerificationResult]::Failed,
+                        "Found credentials in release definition. Variables names: $varList" );
```
Comment (abhaydaga) [inline src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.Release.ps1]: name

File: src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.Release.ps1 (hunk 1 above)
Comment (arvindsingh-cloud) [reply src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.Release.ps1]: changed to evaluate

File: src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.Release.ps1 (hunk 2 above)
Comment (arvindsingh-cloud) [reply src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.Release.ps1]: changed to name

**Pull request merged**